UIKItTemplate/Modules/Notifications/Controller/NotificationsViewController.swift

```diff
@@ -0,0 +1,114 @@
+// NotificationsViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+// Screen with detailed information for зкщвгсеы
```

Comment:
```suggestion
// Screen with detailed information for зкщвгсеы
```
в конце после for не ясно что это

Reply:
Done